src/OpenIddict.Core/OpenIddictProvider.Authentication.cs

```diff
@@ -180,12 +202,17 @@ public partial class OpenIddictProvider<TUser, TApplication> : OpenIdConnectServ
             ticket.SetResources(context.Request.GetResources());
             ticket.SetScopes(context.Request.GetScopes());
 
+            logger.LogDebug("AuthenticationTicket generated succesfully.");
```

Comment:
What about something like `logger.LogDebug("An authentication ticket was successfully generated for user {NameIdentifier}.", principal.FindFirst(ClaimTypes.NameIdentifier)?.Value);`?